src/sentry/lang/javascript/processor.py

```diff
@@ -572,3 +572,9 @@ def populate_source_cache(self, project, frames, release):
                         done_file_list.add(next_filename)
                     else:
                         pending_file_list.add(next_filename)
+
+    def discover_sourcemap(self, result):
+        return discover_sourcemap(result)
```

Comment:
afaik this should cause recursion issues in Python

Granted I'm only half-awake so maybe I'm just not seeing something.